apps/web/src/design-system/icons/general/ErrorIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+/* eslint-disable */
+export function ErrorIcon(props: React.ComponentPropsWithoutRef<'svg'>) {
+  return (
+    <svg width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
    <svg width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```
That way you could control size or color. So the values will be passed to this icon component :)